src/app/researcher/study/[studyId]/review/job-results-status-message.tsx

```diff
@@ -46,15 +46,24 @@ const Errored: FC<ErroredProps> = ({ isApproved, isRejected, jobId }) => {
         message =
             'The code errored out! While logs are not available at this time, consider re-submitting an updated study code.'
     }
+
+    if (!message) {
+        return <Text>Study results will become available once the data organization reviews and approves them.</Text>
+    }
+
     return (
         <>
-            {message && <Text>{message}</Text>}
-            <Group justify="flex-start" align="center">
-                <Text size="xs" fw="bold">
-                    Job ID:
-                </Text>
-                <CopyingInput value={jobId} tooltipLabel="Copy" />
-            </Group>
+            {message && (
```

Comment:
you know message is defined here because otherwise it would have returned earlier